code/modules/clothing/spacesuits/ert_hardsuits.dm

```diff
@@ -191,7 +191,7 @@
 
 /obj/item/clothing/suit/space/hardsuit/ert/paranormal
 	name = "paranormal response team suit"
-	desc = "Powerful wards are built into this hardsuit, protecting the user from all manner of paranormal threats."
+	desc = "A specially designed combat hardsuit produced by Citadel Armories for engaging paranormal threats, this suit is inlaid with interlocking geometric warding sigils and uses a unique holy water liquid-cooling system."
```

Comment:
Holy-water cooling is a bit silly : you need radiators to take the heat out, in this system, water just replaces the fans. However, you wouldn't want to use this system  in space for 2 reasons : 
1) No matter in space means that you can only radiate the heat off, which is rather inefficient.
2) Assuming our space is cold and you _can_ exchange heat, radiators mean that your suit isn't thermally insulated, your suit will thus quickly turn into a popsicle, which is bad unless you're a drask.

Perhaps replace the holy water with a blessed relic that absorbs heat like crazy ? Something like a censer's head that just radiates inexplicably cold vapour ? 
Otherwise, the warding sigils are very cool.

Reply:
I feel like an anomalous object that magically absorbs heat is even less believable than a liquid cooling system, personally.

Reply:
Real life space suits do actually use liquid cooling. Astronauts wear a cooling suit under their hardsuit that uses active liquid cooling to draw heat away from their body and is sent to the hardsuit life support system via heat exchanger.

It can also be used to reject heat into space via evaporative cooling, which allows massive amounts of thermal energy to be dumped in a short period compared to radiation.

Reply:
> I feel like an anomalous object that magically absorbs heat is even less believable than a liquid cooling system, personally.

I understand your concern, but to be fair, this is the _paranormal_ ERT's armour. Using anomalous material to work in dangerous areas where the laws of physics may not apply sounds believable to me. It would become an issue were it to be widespread. But this is the elite theologian force that fight eldritch cults with guns and obsidian rods.

Reply:
TBH I personally think the holy water cooling system fits more without being blatantly anomalous like a magic relic would be.